cmd/billing/billing.go

```diff
@@ -0,0 +1,197 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package billing
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	ybmAuthClient "github.com/yugabyte/ybm-cli/internal/client"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+var BillingCmd = &cobra.Command{
+	Use:   "billing",
+	Short: "Manage billing operations for the account in YugabyteDB Managed",
+	Long:  "Manage billing operations for the account in YugabyteDB Managed",
+	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Help()
+	},
+}
+
+var listUsageCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List billing usage data available for the account in YugabyteDB Managed",
+	Long:  "List billing usage data available for the account in YugabyteDB Managed",
+	Run: func(cmd *cobra.Command, args []string) {
+		authApi, err := ybmAuthClient.NewAuthApiClient()
+		if err != nil {
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+		authApi.GetInfo("", "")
+		startDate, _ := cmd.Flags().GetString("start")
```

Comment:
Again to be honest I would avoid calling the api if the end<start date, a single test should be enough

Reply:
Done, added checks for date validation